### PR TITLE
Helm: set new attribute

### PR DESCRIPTION
**Pull request by franviera92**
I would like to set the following values with helm, I see that it can be set with kustomize but not with helm

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@xpuska513 can you review this contribution?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: charts/airbyte/values.yaml [hunk 1]
```diff
@@ -819,6 +819,11 @@ worker:
     enabled: true
     image: ""
 
+  ## current no exist documentations
+  activity_max_attempt: ""
+  activity_initial_delay_between_attempts_seconds: ""
+  activity_max_delay_between_attempts_seconds: ""
```
[the remark this answers is not in the document]
Comment (franviera92) [reply charts/airbyte/values.yaml]: Ready man

[the remark this answers is not in the document]
Comment (franviera92) [reply charts/airbyte/values.yaml]: @xpuska513 can you review this contribution please?

**Review by xpuska513 — APPROVED** [review]
LGTM

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**